### PR TITLE
docs: migrate README, CONTRIBUTING and LICENSE files

### LINKED ISSUE #19: Migrate and Rewrite the docs to match the new project configuration
This ticket only includes migrating the Readme. 

The migration of the documentation to Docusauros will wait until a following ticket.

### PR DESCRIPTION
**Pull request by NachoVazquez**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

README, CONTRIBUTORS and LICENSE docs aren't yet migrated.

Fixes #19 

## What is the new behavior?

README, CONTRIBUTORS and LICENSE are migrated

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by LayZeeDK — CHANGES_REQUESTED** [review]
question: Why are we changing CI in a PR adding Markdown documents? Is it breaking the build because no projects are affected by the change? What went wrong in the original setup?

**Comment by NachoVazquez** [comment]
> question: Why are we changing CI in a PR adding Markdown documents? Is it breaking the build because no projects are affected by the change? What went wrong in the original setup?

Yes, that's exactly what is happening. It is failing because we didn't build the library before (it wasn't affected) and when trying to copy the library bundle it is not finding it.

File: apps/lumberjack-app/project.json [hunk 1]
```diff
@@ -84,6 +84,7 @@
     },
     "use-lumberjack-bundle": {
       "executor": "nx:run-commands",
+      "dependsOn": ["^build"],
```
Comment (LayZeeDK) [inline apps/lumberjack-app/project.json]: suggestion: Use explicit configuration to make it clearer.
```suggestion
      "dependsOn": [{ "projects": "dependencies", "target": "build" }],
```

Comment (LayZeeDK) [inline apps/lumberjack-app/project.json]: praise: Nice find!

**Review by LayZeeDK — CHANGES_REQUESTED** [review]

File: apps/lumberjack-app/project.json (hunk 1 above)
Comment (NachoVazquez) [reply apps/lumberjack-app/project.json]: Thank you, yeah the result was much more clear!

**Review by LayZeeDK — APPROVED** [review]

**Pull request merged**